notifier.go

```diff
@@ -33,23 +33,33 @@ func New(rawData ...interface{}) *Notifier {
 // Notify sends an error to Bugsnag. Any rawData you pass here will be sent to
 // Bugsnag after being converted to JSON. e.g. bugsnag.SeverityError, bugsnag.Context,
 // or bugsnag.MetaData.
-func (notifier *Notifier) Notify(err error, rawData ...interface{}) (e error) {
-	config := notifier.Config
-	return notifier.NotifySync(err, config.Synchronous, rawData...)
+func (notifier *Notifier) Notify(rawData ...interface{}) (e error) {
+	return notifier.NotifySync(append(rawData, notifier.Config.Synchronous)...)
 }
 
 // NotifySync sends an error to Bugsnag. The synchronous parameter specifies
 // whether to send the report in the current context. Any rawData you pass here
 // will be sent to Bugsnag after being converted to JSON. e.g.
 // bugsnag.SeverityError,  bugsnag.Context, or bugsnag.MetaData.
-func (notifier *Notifier) NotifySync(err error, synchronous bool, rawData ...interface{}) (e error) {
-	event, config := newEvent(errors.New(err, 1), rawData, notifier)
+func (notifier *Notifier) NotifySync(rawData ...interface{}) (e error) {
```

Comment:
It might be worth adding an example for this function alongside `Notify` and `AutoNotify`.

Reply:
I'm a bit hesitant to do so, but can be convinced if you feel strongly about it. This method exists on the `Notifier` (and not globally, on `bugsnag.whatever`) and so would require more in-depth knowledge of how bugsnag-go works than we might want to advertise. Especially for a feature we don't anticipate will be used a lot; If developers need synchronous reporting they are free to call `bugsnag.Configure` with a struct containing `Synchronous: true`.

Reply:
👍